src/js/ripple/server.js

```diff
@@ -434,6 +449,11 @@ Server.prototype.connect = function() {
         log.info(self.getServerID(), 'onerror:',  e.data || e);
       }
 
+      if (Server.TLS_ERRORS.indexOf(e.message) !== -1) {
```

Comment:
_.contains 

would have prevented the bug in the previous version of this pull request and it's more semantic  (easier to read)